paper.tex

```diff
@@ -1109,84 +1109,110 @@ \section*{ARG simplification}
 \vspace{5em}
 \includegraphics[width=\linewidth]{illustrations/simplification}
 \caption{\label{fig-simplification}
-ARG simplification. (A) An ARG simulated from a diploid Wright-Fisher
-model. Note the complex full-sib relationship between B and C; this
-could not be directly represented in a Griffiths-like EARG encoding
-(see section XXX).
+ARG simplification. In all cases the graph visualization is shown on the left
+and the equivalent marginal trees on the right.
+(A) A gARG simulated from a diploid Wright-Fisher
+model. Note that the genome \textsf{K} has three children, and the genomes
+\textsf{E} and \textsf{G} simultaneously have multiple parents and multiple children;
+neither could not be directly represented in a Griffiths-like eARG encoding
+(see section XXX). The penultimate local tree is highlighted to aid discussion (see section YYY).
 (B) Simplified to remove all
 % trying out this terminology - can define in the text
-1-connected graph components (e.g., diamonds).
-(C) Remove nodes that are unary in all local trees.
-(D) Rewrite edges to skip any unary nodes in local trees.
-(E) The local trees for ARG (A).
-(F) The local trees for ARG (D).
+1-connected graph components (e.g., diamonds such as \textsf{JLMN}).
+(C) Remove nodes that are unary in all local trees. This removes both recombinant nodes
+that have only one child (e.g. \textsf{N}) and nodes that have more
+than one child (i.e. are ``common ancestors'') but never represent coalescences
+in the local trees (e.g. \textsf{R}).
+(D) Rewrite edges to bypass any unary nodes in local trees. Note that this loses lineage
+information such as the fact that \textsf{K} is connected to \textsf{A} via \textsf{E}; it
+can also create many extra roots.
 }
 \end{figure}
 
-% Q: would it help to talk about ``tree vertice'' here or would it
+% Q: would it help to talk about ``tree vertices'' here or would it
 % just be more confusing? I worry that people would think that
 % they are two different things then, and it's really important
 % that they get that a node-is-a-node
+
+
 The ideas of ``meaningful'' local tree topologies revolve around the
 presence of ``unary'' nodes: a node in the tree that has exactly
-one child. For example, Fig.~\ref{fig-simplification}E shows the
-local trees along the genome for a simulated ARG
-(Fig.~\ref{fig-simplification}A). Consider the right-most tree,
-and the path above node $D$. Before $D$ meets its common ancestor
-over the interval $(93, 100]$ with the other samples it passes through
-nodes $H, K, N, R$ and $S$, reflecting the corresponding
-traversal through the ARG (see section XXX). Such unary nodes
-in a local tree are not usually considered significant,
+one child. Fig.~\ref{fig-simplification} show a series of successive steps of
+simplification, starting with a complete gARG simulated under a Wright-Fisher
+model, shown both as a mathematical graph on the left, and the equivalent set of 6
```

Comment:
```suggestion
model, shown both as a  graph on the left, and the equivalent set of 6
```
No reader who has got to this point needs the "mathematical" (and including it would probably confuse them)